test/test_helper.exs

```diff
@@ -1 +1,70 @@
 ExUnit.start()
+
+
+defmodule ThriftTestHelpers do
+  @root_dir "test/fixtures/thrift/"
+
+  defmacro __using__(_) do
+    quote do
+      require ThriftTestHelpers
+      import ThriftTestHelpers
+    end
+  end
+
+  def build_thrift_file(base_dir, {file_name, contents}) do
+    file_relative_path = file_name
+    |> Atom.to_string
+
+
+    file_path = Path.join(base_dir, file_relative_path)
+
+    file_path
+    |> Path.dirname
+    |> File.mkdir_p!
+
+    File.write!(file_path, contents)
+    file_relative_path
+  end
+
+  def tmp_dir do
```

Comment:
that doesn't reliable create a temporary directory though. The idea behind this is that each test makes its own directory and puts thrift files into it. That makes it easy to clean up.